test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -76,6 +214,325 @@ var _ = Describe("[performance] Cgroups and affinity", Ordered, func() {
+				Expect(err).ToNot(HaveOccurred())
+
+				// Verify ovs-vswitchd and ovsdb-server process affinity is updated
+				for _, pid := range pidList {
+					cmd := []string{"/bin/bash", "-c", fmt.Sprintf("grep Cpus_allowed_list /proc/%s/status | awk '{print $2}'", pid)}
+					cpusOfovServices, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
+					testlog.Infof("cpus used by ovs service %s is %s", pid, cpusOfovServices)
+					Expect(cpusOfovServices == ovnContainerCpus).To(BeTrue(), "affinity of ovn kube node pods(%s) do not match with ovservices(%s)", ovnContainerCpus, cpusOfovServices)
+					if err != nil {
+						testlog.Error(err.Error())
+					}
+				}
+
+				// Delete testpod2
+				deleteTestPod(testpod2)
+			})
+
+			It("[test_id:64103] ovs process affinity still excludes guaranteed pods after reboot", func() {
+
+				// create a deployment to deploy gu pods
+				dp := newDeployment()
+				testNode := make(map[string]string)
+				testNode["kubernetes.io/hostname"] = workerRTNode.Name
+				dp.Spec.Template.Spec.NodeSelector = testNode
+				err := testclient.Client.Create(context.TODO(), dp)
+				Expect(err).ToNot(HaveOccurred(), "Unable to create Deployment")
+
+				defer func() {
+					// delete deployment
+					testlog.Infof("Deleting Deployment %v", dp.Name)
+					err := testclient.Client.Delete(context.TODO(), dp)
+					Expect(err).ToNot(HaveOccurred())
+				}()
+
+				// Check the cpus of ovn kube node pods and ovs services
+				ovnContainerCpus, err := getOvnContainerCpus(workerRTNode)
+				Expect(err).ToNot(HaveOccurred(), "Unable to fetch cpus of the containers inside ovn kubenode pods")
+				testlog.Infof("cpus used by ovn kube node pods %v", ovnContainerCpus)
+				pidList, err := getOVSServicesPid(workerRTNode)
+				Expect(err).ToNot(HaveOccurred())
+				for _, pid := range pidList {
+					cmd := []string{"/bin/bash", "-c", fmt.Sprintf("grep Cpus_allowed_list /proc/%s/status | awk '{print $2}'", pid)}
+					cpusOfovServices, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
+					testlog.Infof("cpus used by ovs service %s is %s", pid, cpusOfovServices)
+					Expect(cpusOfovServices == ovnContainerCpus).To(BeTrue(), "affinity of ovn kube node pods(%s) do not match with ovservices(%s)", ovnContainerCpus, cpusOfovServices)
+					if err != nil {
+						testlog.Error(err.Error())
+					}
+				}
+
+				testlog.Info("Rebooting the node")
+				// reboot the node, for that we change the numa policy to best-effort
+				// Note: this is used only to trigger reboot
+				policy := "best-effort"
+				// Need to make some changes to pp , causing system reboot
+				// and check if activation files is modified or deleted
+				profile, err = profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
+				Expect(err).ToNot(HaveOccurred(), "Unable to fetch latest performance profile")
+				currentPolicy := profile.Spec.NUMA.TopologyPolicy
+				if *currentPolicy == "best-effort" {
+					policy = "restricted"
+				}
```

Comment:
same here regarding reboot, if the goal is to trigger a system reboot and not a PP specific update, then I'd do a system reboot on the node on which the pod runs on. This 1. won't need a revert 2. keep the tests running on consistent configuration (unless intended the opposite) 3. less dependency by directly rebooting the node instead of w/a.

Reply:
As mentioned above using systemctl reboot or rebooting the node from a pod will cause  unspecified behavior.

Reply:
I think i have addressed this already on why doing system reboot is a bad idea.